open-api/rest-catalog-open-api.py

```diff
@@ -854,6 +854,16 @@ class ContentFile(BaseModel):
 
 class PositionDeleteFile(ContentFile):
     content: Literal['position-deletes']
+    content_offset: Optional[int] = Field(
```

Comment:
What about `referenced_data_file`?

Reply:
I left out `referenced_data_file` because it is unnecessary. Position delete files are sent back through server-side planning as a list. The file scan tasks in the response include a list of indexes into that list to associate files with the deletes that must be applied to them. Adding `referenced_data_file` would duplicate that information and would be confusing, so I think it is better to omit it.

Note that if we were to add fine-grained commits to the update API, we would need to be able to pass it _to_ the service. We can add that later if/when needed.